webclient/impls/pentaho/src/main/javascript/cdf-dd-base.js

```diff
@@ -534,8 +534,8 @@ var SaveRequests = {
     $.post(wd.cde.endpoints.getPluginUrl() + "syncronizer/syncronizeDashboard", saveSettingsParams, function(result) {
       try {
         if(result && result.status == "true") {
-          myself.setDashboardWcdf(wcdf);
-          refreshTitle(wcdf.title);
+          myself.setDashboardWcdf(result.result.wcdf);

```

Comment:
do we really need to use result.result? can't we  have just one result?

Reply:
We can change the callback function parameter name, but the object we receive will still be `{result: ..., status: ...}`. The `result.result` naming is used in many callback functions inside this file so maybe we can leave it like this for consistency.